django/db/backends/base/schema.py

```diff
@@ -201,6 +202,13 @@ def effective_default(self, field):
                 default = six.binary_type()
             else:
                 default = six.text_type()
+        elif getattr(field, 'auto_now', False) or getattr(field, 'auto_now_add', False):
+            default = datetime.now()
```

Comment:
If we don't and `USE_TZ=True` the default datetime used on column creation will be in the system timezone instead of UTC like Django assumes datetime are stored in the database.